Fixed grammatical mistakes and typos in Tutorial 6 (#1842)
cyberbotics/webots: docs/guide/tutorial-6-4-wheels-robot.md

## Tutorial 6: 4-Wheeled Robot (60 Minutes)

This tutorial aims at creating your first robot from scratch.
The robot will consist of a body, four wheels, and two distance sensors.

> The result is depicted in [this figure](#3d-view-of-the-4-wheels-robot-note-that-the-coordinate-system-representations-of-the-robot-body-and-of-its-wheels-are-oriented-the-same-way-their-px-vector-defines-the-left-of-the-robot-their-py-vector-defines-the-top-of-the-robot-and-their-pz-vector-defines-the-front-of-the-robot-the-distance-sensors-are-oriented-in-a-different-way-their-px-vector-indicates-the-direction-of-the-sensor).

Reviewer: ```suggestion
The result is depicted in [this figure](3d-view-representation-of-the-4-wheeled-robot-note-that-the-coordinate-system-representations-of-the-robot-body-and-its-wheels-are-oriented-the-same-way-the-px-vector-in-red-defines-the-left-of-the-robot-the-y-vector-in-green-defines-the-top-of-the-robot-and-their-z-vector-in-blue-defines-the-front-of-the-robot-the-distance-sensors-are-oriented-differently-their-x-vector-indicates-the-direction-of-the-sensor).
```

> The [next figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices) shows the top view of the robot.

Reviewer: ```suggestion
The [next figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device) shows the top view of the robot.
```

- The set containing the [Solid](../reference/solid.md) node and all its derived nodes are called the *solid nodes*.

- A similar definition is applied for the [Device](../reference/device.md), [Robot](../reference/robot.md), [Joint](../reference/joint.md) and [Motor](../reference/motor.md) nodes.
You can get more information about the node hierarchy in the [nodes chart diagram](../reference/node-chart.md).

- Most sensors and actuators are both [Solid](../reference/solid.md) and [Device](../reference/device.md) nodes at the same time.

- The main structure of a [Robot](../reference/robot.md) model is a tree of [Solid](../reference/solid.md) nodes linked together.
The root node of this tree should be a [Robot](../reference/robot.md) node.

- The solids are linked together by [Joint](../reference/joint.md) nodes.
A [Device](../reference/device.md) node should be the direct child of either a [Robot](../reference/robot.md) node, a [Solid](../reference/solid.md) node or a [Joint](../reference/joint.md) node.

- A [Joint](../reference/joint.md) node is used to add one (or two) degree(s) of freedom (DOF), between its parent and its child.

> The direct parent and child of a [Joint](../reference/joint.md) nodes are both [Solid](../reference/solid.md) nodes.

Reviewer 2: I believe the singular form was correct here.
```suggestion
The direct parent and child of a [Joint](../reference/joint.md) node are both [Solid](../reference/solid.md) nodes.
```

> Please refer to [this figure](#high-level-representation-of-the-4-wheels-robot).

Reviewer: ```suggestion
Please refer to [this figure](#high-level-representation-of-the-4-wheeled-robot).

Add a [Shape](../reference/shape.md) node containing a Box geometry to the [Robot](../reference/robot.md) node.
Set the color of the [Shape](../reference/shape.md) to red.
Use the [Shape](../reference/shape.md) to define the `boundingObject` field of the [Robot](../reference/robot.md) node.
The dimension of the box is `(0.1, 0.05, 0.2)`.
Add a [Physics](../reference/physics.md) node to the [Robot](../reference/robot.md).

> The [figure](#low-level-representation-of-the-4-wheels-robot) represents all the nodes defining the robot.

Reviewer: ```suggestion
The [figure](high-level-representation-of-the-4-wheeled-robot) represents all the nodes defining the robot.
```

For the first wheel, the [Solid](../reference/solid.md) translation should be defined to `(0.06, 0, 0.05)` in order to define the relative gap between the body and the wheel, and the rotation to `(0 0 1 1.5708)` for the wheel cylinder to be correctly oriented.
The [HingeJointParameters](../reference/hingejointparameters.md) anchor should also be defined to `(0.06, 0, 0.05)` to define the rotation origin (relatively to the body).
Finally, the [HingeJointParameters](../reference/hingejointparameters.md) axis should define the rotation axis.
In our case it's along the x-axis (so `(1, 0, 0)`).

> **Hands on #3**: Add a [HingeJointParameters](../reference/hingejointparameters.md) node, and enter the field values as described above.
Some signs obviously have to be updated for other wheels.

Now, let's implement the cylinder shape of the wheels.

> > **Hands on #4**: Complete the missing nodes to get the same structure as the one depicted in [this figure](#low-level-representation-of-the-4-wheels-robot).

Reviewer: ```suggestion
> **Hands on #4**: Complete the missing nodes to get the same structure as the one depicted in [this figure](#low-level-representation-of-the-4-wheeled-robot).
```

### Sensors

The sensors used on the e-puck of [Tutorial 4](tutorial-4-more-about-controllers.md) are not the same as the ones used in this section. Pay attention to their measuring interval which is between `0 cm = 0` and `10 cm = 1000`.
Find more information about the `lookupTable` field [here](../reference/distancesensor.md#lookup-table).

The last part of the robot modeling is to add the two distance sensors to the robot.
This can be done by adding two [DistanceSensor](../reference/distancesensor.md) nodes as direct children of the [Robot](../reference/robot.md) node.
Note that the distance sensor acquires its data along the +*x*-axis.

> So, it is necessary to rotate the distance sensor to point their x-axis outside the robot (see the [figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices)).

Reviewer: ```suggestion
So, it is necessary to rotate the distance sensor to point their x-axis outside the robot (see the [figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device)).
```

%end

> **Hands on #5**: Add the two distance sensors as explained above.
The distance sensors are at an angle to 0.3 [rad] with the front vector of the robot.
Set thei graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
Set thei color to blue.

> Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).

Reviewer: ```suggestion
Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device).
```

Author: Oh yes, I see my mistake here. Thanks a lot for pointing them out :)

Reviewer: You're welcome, thank you for your help making the doc better.

> Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheeled-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device).

Reviewer 2: "thei" seems wrong, should be "the" I guess?

Author: oh yes, my bad
